CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 4.0.1
+  - Added threadsafety; this plugin can now be used in pipelines with more than one worker.
```

Comment:
Please consider adding the PR link (MD format). The link is helpful in the changelog, but is especially helpful when we generate release notes.